api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -303,7 +306,25 @@ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 SELECT 0, c.concept_id, c.concept_synonym_name
 FROM \`$BQ_PROJECT.$BQ_DATASET.concept_synonym\` c"
 
-
+###########################
+# Domain_Vocabulary_Info #
+###########################
+echo "Updating all concept count in domain_vocabulary_info"
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.domain_vocabulary_info\`
+(concept_id,vocabulary_id,all_concept_count,standard_concept_count)
+with all_concept_count as
+(select d.domain_concept_id as concept_id,c.vocabulary_id as vocabulary_id,count(distinct c.concept_id) as all_concept_count
```

Comment:
So I would do this by joining to domain_info, with a single select, similar to:

https://github.com/all-of-us/workbench/pull/1255/files#diff-f8104808aff893885bceb08ea8f80fd3R219

insert into...

select d.concept_id as concept_id, c.vocabulary_id as vocabulary_id, count(distinct c.concept_id) as all_concept_count, SUM(CASE WHEN c.standard_concept IN ('S', 'C') THEN 1 ELSE 0 END) as standard_concept_count from
...domain_info d JOIN ... concept

Reply:
Ahh, concept_id is for domain ...  I thought that was the vocabulary_concept_id .. I would use domain_id as the key here. The concept table has domain_id and vocabulary_id ... If we do keep concept_id here it should be the vocabulary_concept_id ... or we could have both. But they are both in the vocabulary and domain tables when we want them anyway.

Reply:
The pk here should be domain_id, vocabulary_id in plain english :)

Reply:
So if you want to change that, you'll need to change the primary key of domain_info (which means switching to a new table.) Right now its primary key is concept ID.

We could change this to domain_id in a new table, but I'm not sure it really matters?

Reply:
Sorry i missed this one in review. domain_id and vocab_id should be the pks because concept_id's can change in omop updates. . They are the pk's of the domain and vocabulary table ..  I think we should be consistent.  

Why does it mean switching to new table?

Reply:
You can't change the primary key of a table in MySQL AFAIK. We'd need to drop and recreate it.

Reply:
I'll also note that if the concept ID did change, we'd have to change the CDR metadata in domain_info to match anyway. So while I agree it may be more consistent, not sure it really matters?

Reply:
@biopete do you still feel like we need to address this?